aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/SyncProcessor.java

```diff
@@ -340,4 +380,25 @@ public SyncProcessor build() {
             return modelSchemaRegistry.getModelSchemaForModelInstance(modelWithMetadata.getModel());
         }
     }
+
+    private static final class ModelSyncMetrics {
```

Comment:
Does this get sent back to user? Does it need to be public, with equals/hashCode/toString and field accessors?

Reply:
Actually...this can be removed entirely.